Sources/LogtoClient/LogtoAuthSession/LogtoWebViewAuthSession.swift

```diff
@@ -31,12 +32,14 @@ public class LogtoWebViewAuthSession: NSObject {
 
     let uri: URL
     let redirectUri: URL
+    let socialPlugins: [LogtoSocialPlugin]
     let onFinish: FinishHandler
     var viewController: LogtoWebViewAuthViewController?
 
-    public init(_ uri: URL, redirectUri: URL, onFinish: @escaping FinishHandler) {
+    public init(_ uri: URL, redirectUri: URL, socialPlugins: [LogtoSocialPlugin], onFinish: @escaping FinishHandler) {
         self.uri = uri
         self.redirectUri = redirectUri
+        self.socialPlugins = socialPlugins
```

Comment:
Should handle these uncovered code lines?

<img width="665" alt="image" src="https://user-images.githubusercontent.com/12833674/161239708-8924e990-2944-428b-8d18-c625f43bb7a5.png">

Reply:
will add tests later LOG-1525